### ISSUE TITLE
[Package Issue]: AnalogDevices.LTspice

### ISSUE THREAD

**Issue by benjibst** [microsoft/winget-pkgs#125568]
### Please confirm these before moving forward

- [X] I have searched for my issue and not found a work-in-progress/duplicate/resolved issue.
- [X] I have not been informed if the issue is resolved in a preview version of the winget client.

### Category of the issue

Installer hash mismatch.

### Brief description of your issue

hash of file when downloaded from website: 
6718ABBD4F4FAEFE375C579DE775151525A9A1CCA821041796588A6EC05A5C99
hash of file in package manifest: 
48998B09194BF2D56AAE3D36810BB3AE536E7C60FBA76A30DE78107742E5EA7A

### Steps to reproduce

try to download lt spice through winget
throws error Hash value of installer does not coincide

### Actual behavior

winget throws error

### Expected behavior

winget doesnt throw error

### Environment

```raw
Windows-Paket-Manager v1.6.2771
Copyright (c) Microsoft Corporation. Alle Rechte vorbehalten.

Windows: Windows.Desktop v10.0.19045.3570
Systemarchitektur: X64
Paket: Microsoft.DesktopAppInstaller v1.21.2771.0

WinGet-Verzeichnisse
-----------------------------------------------------------------------------------------------------------------------
Protokolle                                   %LOCALAPPDATA%\Packages\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\Local…
Benutzereinstellungen                        %LOCALAPPDATA%\Packages\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\Local…
Verzeichnis für portierbare Links (Benutzer) %LOCALAPPDATA%\Microsoft\WinGet\Links
Portables Linkverzeichnis (Computer)         C:\Program Files\WinGet\Links
Portierbarer Paketstamm (Benutzer)           %LOCALAPPDATA%\Microsoft\WinGet\Packages
Portierbarer Paketstamm                      C:\Program Files\WinGet\Packages
Portierbares Paketstamm (x86)                C:\Program Files (x86)\WinGet\Packages
Installationsprogrammdownloads               %USERPROFILE%\Downloads

Links
-----------------------------------------------------------------------------------------
Datenschutzerklärung              https://aka.ms/winget-privacy
Lizenzvereinbarung                https://aka.ms/winget-license
Hinweise von Drittanbietern       https://aka.ms/winget-3rdPartyNotice
Startseite                        https://aka.ms/winget
Windows Store-Nutzungsbedingungen https://www.microsoft.com/en-us/storedocs/terms-of-sale

Administratoreinstellung                  Status
-----------------------------------------------------
LocalManifestFiles                        Deaktiviert
BypassCertificatePinningForMicrosoftStore Deaktiviert
InstallerHashOverride                     Deaktiviert
LocalArchiveMalwareScanOverride           Deaktiviert
```

**Comment by Dragon1573** [comment]
@wingetbot is currently working on this issue.

- https://github.com/microsoft/winget-pkgs/pull/125589

**Comment by mdanish-kh** [comment]
Close with reason: Resolved in #126192;
